schemas/blocks/list_collector.json

```diff
@@ -18,6 +18,35 @@
     "required": ["id", "value"],
     "additionalProperties": false
   },
+  "summary": {
+    "type": "object",
+    "properties":{
+      "type": {
+          "type": "string",
+          "enum": ["ListCollectorSummary"]
+      },
+      "item_title": {
```

Comment:
For `item_title` we should be able to reuse the `#/string_with_placeholders"` definition?

https://github.com/ONSdigital/eq-schema-validator/blob/refactor-list-collector-summary/schemas/string_interpolation/definitions.json#L3-L26